react/components/Table/totalizers/TotalizerValue.js

```diff
@@ -0,0 +1,36 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import config from 'vtex-tachyons/config.json'
+
+import Spinner from '../../Spinner'
+
+class TotalizerValue extends PureComponent {
+  render() {
+    const {
+      item: { value, isLoading },
+    } = this.props
+
+    if (isLoading) {
+      return (
+        <div className="mt2">
+          <Spinner
+            size="14"
+            color={config.semanticColors.background['base--inverted']}
+          />
+        </div>
+      )
+    }
+
+    if (value) {
```

Comment:
Invert this, the last case should be the default case.

```js
if (value == null) {
  return null
}
```

Also it should test if it's null or undefined like above, because if the value is zero it would return null.